packages/x-components/src/x-installer/x-installer/types.ts

```diff
@@ -79,3 +79,19 @@ export interface ExtraPluginsOptions {
  * @public
  */
 export type VueConstructorPartialArgument = Partial<ConstructorParameters<VueConstructor>[0]>;
+
+/**
+ * Interface for the returned type of the XInstaller.init function.
+ *
+ * @public
+ */
+export interface InitWrapper {
+  /** The Vue component used as root of the application. */
```

Comment:
This is not the component, but the instance of the component

```suggestion
  /** The root Vue instance of the application. */
```